lib/ansiblelint/rules/PackageHasRetryRule.py

```diff
@@ -12,7 +12,7 @@ class PackageHasRetryRule(AnsibleLintRule):
         'network communication and the availability of remote '
         'servers. To mitigate the potential problems, retries '
         'should be used via '
-        '``register: my_result`` and ``until: my_result | success``'
+        '``register: my_result`` and ``until: my_result is success``'
```

Comment:
Should be `is successful` (I know `is success` works but it's not grammatically correct)

Reply:
@willthames is it succeeded or successful?  Ansible 2.7 docs has it `succeeded`.  https://docs.ansible.com/ansible/latest/user_guide/playbooks_conditionals.html.

Ansible's deprecation warning needs to be updated as well to the newer syntax: `[DEPRECATION WARNING]: Using tests as filters is deprecated. Instead of using result|success use result is success. `

Reply:
https://github.com/ansible/ansible/blob/devel/lib/ansible/plugins/test/core.py#L170-L177 suggests all will work. I'd argue `is successful` is more readable than `is succeeded`. 

The [2.5 porting guide](https://github.com/ansible/ansible/blob/devel/docs/docsite/rst/porting_guides/porting_guide_2.5.rst#jinja-tests-used-as-filters) suggests `successful` as the way forward but I don't disagree that it's inconsistent in both documentation and in deprecation messages.